.github/workflows/odin.yml

```
@@ -0,0 +1,30 @@
+  schedule:
+    - cron: 0 20 * * *
+
+env:
+  FORCE_COLOR: "1"
+
+jobs:
+  check:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, windows-latest] 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)